runtime/test-runtime/client/src/lib.rs

```diff
@@ -321,3 +316,20 @@ pub fn new_light_fetcher() -> LightFetcher {
 pub fn new_native_executor() -> sc_executor::NativeExecutor<LocalExecutor> {
 	sc_executor::NativeExecutor::new(sc_executor::WasmExecutionMethod::Interpreted, None, 8)
 }
+
+/// Extrinsics that must be included in each block.
+pub fn needed_extrinsics() -> (polkadot_test_runtime::UncheckedExtrinsic, polkadot_test_runtime::UncheckedExtrinsic) {
```

Comment:
I think it would be nice to take the `heads` as parameter.

Reply:
And why not just return a `Vec`? If in the future we add another inherent, we would need to touch every line that uses this function.